client/src/apis/MemberApi.ts

```diff
@@ -1,5 +1,9 @@
 import tokenRequestApi from "./TokenRequestApi";
-
+import { useRecoilValue } from "recoil";
+import { LogInState } from "../recoil/atoms/LogInState";
```

Comment:
Log in - 동사
Login - 명사

1이 맞는 표기 같지만 명사로 쓰일때는 놀랍게도 한단어입니다.

isLoggedIn - O
Login - O
```suggestion
import { LogInState } from "../recoil/atoms/LogInState";
import { LoginState } from '../recoil/atoms/loginState';
```